vagrant/provisioning/roles/backup_jar_war/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+- name: ensure Config-Server is stopped
+  become: yes
+  systemd:
+    name: config-server
+    state: stopped
+
+- name: ensure Snowbound is stopped
+  become: yes
+  systemd:
+    name: snowbound
+    state: stopped    
+
+- name: ensure External Portal is stopped
+  become: yes
+  systemd:
+    name: arkcase
+    state: stopped
+  when: foia_portal_context is defined
+
+- name: backup data folders 
+  become: yes
+  file:
+    path: "{{ root_folder }}/backups/webapps"
+    state: present
+
+- name: Config-Server jar backuping
```

Comment:
Why should jar/war files be backed up? They can be restored using the same installation process that deployed them initially.

Reply:
The versions might have changed. We don't have any place where we keep the old version.